model_signing/signature/utils.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Utils is an anti pattern. Should these be moved to closer to where they are used?

Reply:
removed.